valohai_yaml/pipelines/conversion.py

```diff
@@ -102,3 +103,8 @@ def convert_executionlike_node(
         }
 
         return node_data
+
+    def convert_expression(self, expression: ExpressionValue) -> ExpressionValue:
+        if isinstance(expression, list):
+            return ",".join(expression)
```

Comment:
Shouldn't this take `multiple-separator` into account?

Reply:
my bad, it should yes

Reply:
Related discussion: https://valohai.slack.com/archives/C3J44LTL6/p1710758618962599